Winapp2.ini

```diff
@@ -13036,7 +13042,7 @@ LangSecRef=3024
 Detect=HKLM\Software\NVIDIA Corporation
 FileKey1=%AppData%\NVIDIA\*Cache|*.*|RECURSE
 FileKey2=%CommonAppData%|DisplaySessionContainer*.log;DisplaySessionContainer*.log_backup*;NvcDispCorePlugin.log;NvcDispCorePlugin.log_backup*;NVDisplay.ContainerLocalSystem.log;NVDisplay.ContainerLocalSystem.log_backup*;NVDisplayContainerWatchdog.log;NVDisplayContainerWatchdog.log_backup*
-FileKey3=%CommonAppData%\NVIDIA|*.log;*.log_backup1;Resource.old;*.bak|RECURSE
+FileKey3=%CommonAppData%\NVIDIA|*.log;*.log*;Resource.old;*.bak|RECURSE

```

Comment:
The pattern "\*.log" is no longer needed with this change, as it is covered by the pattern "\*.log*".

FileKey3=%CommonAppData%\\NVIDIA|\*.log\*;Resource.old;\*.bak|RECURSE

Reply:
OK It would be nice. I don't know that. Sry.